fetch2/src/main/java/com/tonyodev/fetch2/HttpUrlConnectionDownloader.kt

```diff
@@ -193,6 +193,10 @@ open class HttpUrlConnectionDownloader @JvmOverloads constructor(
     }
 
     override fun getRequestSupportedFileDownloaderTypes(request: Downloader.ServerRequest): Set<Downloader.FileDownloaderType> {
+        if(fileDownloaderType == Downloader.FileDownloaderType.SEQUENTIAL) {
```

Comment:
```suggestion
        return when (fileDownloaderType) {
            Downloader.FileDownloaderType.SEQUENTIAL -> mutableSetOf(fileDownloaderType)
            else -> try {
                getRequestSupportedFileDownloaderTypes(request, this)
            } catch (e: Exception) {
                mutableSetOf(fileDownloaderType)
            }
```

Probably will be more readable